packages/app/src/cli/services/environment/identifiers-extensions.ts

```diff
@@ -37,6 +37,20 @@ export async function ensureExtensionsIds(
     onlyRemoteExtensions = matchResult.onlyRemote
   }
 
+  for (const extension of matchExtensions.toMigrate) {
+    // eslint-disable-next-line no-await-in-loop
+    const confirmed = await migratePrompt(extension.local, extension.remote)
+
+    if (!confirmed) {
+      return err('user-cancelled')
+    }
+
+    // eslint-disable-next-line no-await-in-loop
+    await migrateToUiExtension(extension.remote.id, options.appId)
```

Comment:
Should this not be `options.app.apiKey`?

Reply:
Looking at how this is used elsewhere, and checking it with the env file the CLI saves, the appId is the api key